src/components/resizable_container/resizable_container.tsx

```diff
@@ -68,9 +69,21 @@ export interface EuiResizableContainerProps
    */
   onPanelWidthChange?: ({}: { [key: string]: number }) => any;
   onToggleCollapsed?: ToggleCollapseCallback;
+  /**
+   * Called when resizing starts
+   */
+  onResizeStart?: (trigger: 'pointer' | 'key') => any;
+  /**
+   * Called when resizing ends
+   */
+  onResizeEnd?: () => any;
```

Comment:
Typewise, I think it's more correct for these to return `void` instead of `any` (I know you were probably copying `onPanelWidthChange` above, and I'd be ++ for changing that as well)
```suggestion
  onResizeStart?: (trigger: 'pointer' | 'key') => void;
  /**
   * Called when resizing ends
   */
  onResizeEnd?: () => void;
```